apps/barista/src/layout/smalltile/smalltile.scss

```
@@ -1,46 +1,26 @@
 @import '../../styles/global/variables';
 @import '../../../../../components/core/src/style/colors';
```

We can use the preprocessed styles
```suggestion
@import 'colors';
```